src/_index.scss

```diff
@@ -159,8 +159,13 @@ $warning_color: $BANANA_900;
         $text-size: $text-size * 1px;
     }
 
-    // Workaround GTK clamping instead of rounding up
-    @return ($pixels / $text-size * 1rem) + 0.000000001rem;
+    @if ($pixels > 1) {
+        // Workaround GTK clamping instead of rounding up
+        @return ($pixels / $text-size * 1rem) + 0.000000001rem;
+    } @else {
+        // Workaround GTK clamping instead of rounding up
+        @return ($pixels / $text-size * 1rem) - 0.000000001rem;
+    }
```

Comment:
Am I reading this right? It looks like you're returning the same thing, regardless?

Reply:
No one is `-` and one is `+`. We were always doing `+` before which if you pass in -6 and then +0.00001 that's not the same as +6 and then +0.000001

Reply:
Ah, make sense. Shouldn't this be based around `0`, not `1`, then?

Reply:
Yeah, you're smart and I have no idea what I was thinking lmao